canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -356,7 +355,19 @@ class ToolTip extends React.Component {
 		if (this.props.children) {
 			// when children are passed in, tooltip will handle show/hide, otherwise consumer has to hide show/hide tooltip
 			const mouseover = () => this.setTooltipVisible(true);
-			const mouseleave = () => this.setTooltipVisible(false);
+			// const mouseleave = () => this.setTooltipVisible(false);
```

Comment:
Code that i snot needed should be removed not commented out.  (Note: There shouldn't be any commented out code merged into main unless there is a comment to indicate why the code is commented out.)